src/actions/relay/tbits.js

```diff
@@ -0,0 +1,40 @@
+const { ActionTransport } = require('@microfleet/core');
+const { HttpStatusError } = require('common-errors');
+
+const RelayDisabledError = new HttpStatusError(412, 'tbits relay auth disabled');
+
+/**
+ * @api {amqp} <prefix>.relay.tbits Relay Tbits Authentication
+ * @apiVersion 1.0.0
+ * @apiName RelayTbitsAuth
+ * @apiGroup Relay
+ *
+ * @apiDescription Verifies Tbits Session Id, creates "shadow" user and issues JWT token if everything is in order
+ *
+ * @apiParam (Payload) {String} sessionUid - tbits session uid to verify
+ */
+async function tbitsAuth({ params, log }) {
+  if (!this.config.tbits.enabled) {
+    throw RelayDisabledError;
+  }
+
+  const response = await this.tbits.verify(params.sessionUid);
```

Comment:
uncertain action, in verify you process register and login. 
I think you should add new method to tbits, f.e verifyAndLogin

Reply:
lets call it `authenticate` we technically create a "shadow" user copying data from tbits and issuing our own JWT token.
sort of oauth, but not that secure